source/compat.rst

```diff
@@ -90,3 +90,25 @@ best results, one of the following browsers recommended:
 -  Google Chrome version 36.0+
 
 -  Safari 6+
+
+Notice Of Management OSes and Hypervisors to be Deprecated
+----------------------------------------------------------
+
+The following hypervisors will no longer be tested from the first release AFTER
+1st June 2018. And will be removed from the CloudStack codebase from the first 
+release after 1st November 2018.
+
+-  XenServer 6.2
+-  XenServer 6.5
+-  vSphere 5.0
+-  vSphere 5.1
+-  Ubuntu 12.04
```

Comment:
How is Ubuntu in the Hypervisor list? should it have a notice about 'with KVM'?

Reply:
Actually, Ubuntu 12.04 only applies to baremetal it seems so can be dropped from this list